File: sendmessage.php
<?php header('Content-type: text/html; charset=utf-8')?>
<?php
$sendto   = "kiev.tatuazh@gmail.com";
$phone = $_POST['user_phone'];
$name  = nl2br($_POST['name']);

$subject  = "Call me back";
$headers  = "From: " . strip_tags($phone) . "\r\n";
$headers .= "Reply-To: ". strip_tags($phone) . "\r\n";
$headers .= "MIME-Version: 1.0\r\n";
$headers .= "Content-Type: text/html;charset=utf-8 \r\n";

$msg  = "<html><body style='font-family:Arial,sans-serif;'>";
$msg .= "<h2 style='font-weight:bold;border-bottom:1px dotted #ccc;'>New User Feedback</h2>\r\n";
$msg .= "<p><strong>Sent by:</strong> ".$phone."</p>\r\n";
$msg .= "<p><strong>Message:</strong> ".$name."</p>\r\n";
$msg .= "</body></html>";


if(@mail($sendto, $subject, $msg, $headers)) {
	echo "true";
} else {
	echo "false";
}

?>

File: sendmessage.py
#!/usr/bin/python

#CGI getting POST variables
import cgi
form = cgi.FieldStorage()
name = form["name"]
phone  = form["user_phone"]

#call mail function
mail("olexandr.kara@gmail.com",
   "Перезвоните мне",
   "Имя клиентки: "name+"; Её мобильный: "+phone)


# импорт нужных модулей для работы с почтой
import smtplib
import os

#Ваши данные мейл и пароль от gmail ящика
gmail_user = "kiev.tatuazh@gmail.com"
gmail_password = "karaanna"

# главная функция принимающая 3 параметра
# адресат, тему письма, само сообщение 
def mail(to, subject, text):

# инициализируем наши данные
   msg['From'] = gmail_user
   msg['To'] = to
   msg['Subject'] = subject

# инициализируем smtp сервер и отправляем письмо
   mailServer = smtplib.SMTP("smtp.gmail.com", 587)
   mailServer.ehlo()
   mailServer.starttls()
   mailServer.ehlo()
   mailServer.login(gmail_user, gmail_pwd)
   mailServer.sendmail(gmail_user, to, msg.as_string())
   mailServer.close()
